Adding LICENSE file

bernhold opened the pull request:
We need to get all of the contributing institutions and date ranges for their contributions.

After we add the LICENSE file we should probably also start adding copyright headers to all of the individual files.

bernhold committed:
First draft of LICENSE file

We need to get all of the contributing institutions and date ranges for their contributions.

After we add the LICENSE file we should probably also start adding copyright headers to all of the individual files.

bernhold commented:
Likely other intuitions
* ~ANL 2019 UChicago Argonne, LLC ?~
* U Florida 2020

Who were the following affiliated with at the time of their contributions?
* Christopher Ulmer 2017-2018
* Gerrit vanCoevering 2018


Can't find evidence of actual commits from the following in either svn or git repositories:
* SNL
* Duke U
* Danqing We
* Sam Morris

bernhold committed:
Update LICENSE

Revise ORNL and UTK start year
Add ANL

sblondel commented:
- Christopher Ulmer was Penn State University
- Gerrit Vancoevering was University of Michigan
- SNL and Duke only contributed to a draft of document saved in the repo
- Sam Morris has a fork of the code where he made some changes but it has not been merged and I am not sure when or if it will be merged
- Danqing Wu is from ANL and contributed from November 2014 to September 2016

bernhold committed:
I think we have all of the contributors covered now

PSU and UM are a guess at who owns the copyright.  It might be the Trustees or the Research Foundation or something else.  But I think it is close enough for government work.

bernhold commented:
Thanks @sblondel I think that covers everyone well enough.  So I think this PR is ready to merge.  Sophie/Philip please review and let me know if I need to do anything else here.
